web/src/Root.tsx

```diff
@@ -66,7 +66,9 @@ const ProtectedRoute = ({
 };
 
 const ThemeContext = React.createContext({
-  setThemeState: () => {},
+  setThemeState: (themeState?: ThemeState) => {
+    console.log("setThemeState used before being set", themeState);
```

Comment:
Was this meant to be here? It's fine if so, just wanted to double check.

Reply:
Yeah this is just a placeholder for now. Going to move the themeset stuff into it's own provider hook and i wanted to make sure i remember to add some error handling.